.github/workflows/build.yml

```diff
@@ -86,30 +86,30 @@ jobs:
       with:
         path: 'flambda_backend'
 
-    - name: Cache OCaml 4.12 and dune
+    - name: Cache OCaml 4.14 and dune
       uses: actions/cache@v1
       id: cache
       with:
-        path: ${{ github.workspace }}/ocaml-412/_install
-        key: ${{ matrix.os }}-cache-ocaml-412-dune-341
+        path: ${{ github.workspace }}/ocaml-414/_install
+        key: ${{ matrix.os }}-cache-ocaml-414
```

Comment:
(If the suffix is removed, possibly unpin dune.)

Reply:
Ah this was actually pinned.  Have updated to 3.6.1 which is what I currently get in a 4.14.0 switch by default.